index.js

```diff
@@ -17,6 +17,34 @@ mongoose
   })
   .then(() => {
     // Run your code here, after you have insured that the connection was made
+
+    return Recipe.create(data[0]);
```

Comment:
here you created a recipe with first element of data, but in the Recipe model,, title should be unique. it means in line 27 when you wanna create your database with data file,, you have a duplicate title,, and you get an error,,<